### PR TITLE
Add rook-cephfs storage class for filesystem PVCs

### PR DESCRIPTION
**Pull request by rmohr**
If rook-ceph gets enabled, it can now used for filesystem PVCs.

```
$ kubectl.sh get storageclasses
selecting docker as container runtime
NAME                        PROVISIONER                     RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
local                       kubernetes.io/no-provisioner    Delete          WaitForFirstConsumer   false                  33m
rook-ceph-block (default)   rook-ceph.rbd.csi.ceph.com      Delete          Immediate              true                   31m
rook-cephfs                 rook-ceph.cephfs.csi.ceph.com   Delete          Immediate              false                  31m
```

Also bumping the centos-stream base version, since for the initrd of the older kenerl we don't have kernel modules anymore.

### CODE REVIEW

**Comment by akalenyu** [comment]
Awesome!

The only suggestion I have is that we add a ready check for cephfs similarly to https://github.com/kubevirt/kubevirtci/blob/ab5a0f27c874b85a32e99f2be43fd378b73a5e59/cluster-provision/k8s/1.23/rook-ceph.sh#L23

File: cluster-provision/centos8/version [hunk 1]
```diff
@@ -1 +1 @@
-20210210.0
+20220125.1
```
Comment (xpivarc) [inline cluster-provision/centos8/version:1]: Just from curiosity why is it needed?

**Comment by rmohr** [comment]
> The only suggestion I have is that we add a ready check for cephfs similarly to

Will add it asap :+1:

**Pull request closed without merging**